presto-main/src/main/java/com/facebook/presto/sql/ExpressionUtils.java

```diff
@@ -283,4 +289,62 @@ public Expression rewriteLambdaExpression(LambdaExpression node, Void context, E
             }
         }, expression);
     }
+
+    public static GroupingElement removeGroupingElementPrefix(GroupingElement groupingElement)
```

Comment:
This util class is deprecated. Looking for a better place for these methods.

Reply:
@highker do you know what would be the alternative place for parser dependent utilities?